pkg/controller/provider/controller.go

```diff
@@ -232,6 +232,13 @@ func (r Reconciler) Reconcile(request reconcile.Request) (result reconcile.Resul
 		result.RequeueAfter = base.SlowReQ
 	}
 
+	// Stop reconciliation when auth fails
+	if provider.Status.HasCondition(ConnectionAuthFailed) {
+		result.RequeueAfter = 0
+		err = nil
+		return
```

Comment:
With PR https://github.com/konveyor/forklift-controller/pull/453 I can move this check right after the validation.
 https://github.com/konveyor/forklift-controller/blob/367d2e9e2378ba7419881aff80fd32472a4b8538/pkg/controller/provider/controller.go#L186-L189